src/core/tests/constant.cpp

```diff
@@ -1373,6 +1373,52 @@ TEST(constant, float16_vector) {
     EXPECT_EQ(p[3], float16(0));
 }
 
+TEST(constant, float8_e4m3_vector) {
+    const auto data_vec = std::vector<ov::float8_e4m3>{std::numeric_limits<ov::float8_e4m3>::lowest(),
+                                                       -std::numeric_limits<ov::float8_e4m3>::min(),
+                                                       std::numeric_limits<ov::float8_e4m3>::min(),
+                                                       std::numeric_limits<ov::float8_e4m3>::max(),
+                                                       std::numeric_limits<ov::float8_e4m3>::denorm_min(),
+                                                       -1.5f,
+                                                       -1.f,
+                                                       -0.5f,
+                                                       0.f,
+                                                       0.5f,
+                                                       1.f,
+                                                       1.5f};
+    Shape data_shape{data_vec.size()};
+    EXPECT_EQ(data_vec.size(), shape_size(data_shape));
```

Comment:
Is it possible otherwise? If that's crucial for the rest of the test use `ASSERT_EQ`.

Reply:
For this specific test case it seems to be always true, `ASSERT_EQ` could be used as well, or even this test check is redundant. Will consider update within following PRs. Thanks!